### PR TITLE
Initial integration of SOME/IP transport

### PR DESCRIPTION
**Pull request by neelam-kushwah**
#35

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -55,6 +55,15 @@ This step is essential for establishing a connection between the emulator and th
 +
 image::screenshots/zenoh_router.png[]
 
+=== SOME/IP
+* Install vsomeip, refer to this https://github.com/eclipse-uprotocol/up-transport-vsomeip-python?tab=readme-ov-file#prerequisites[link] for the installation instructions.
+* Once vsomeip is installed, execute the following command to install up-transport-vsomeip-python.
++
+[source]
+----
+python3 -m pip install 'up-transport-vsomeip-python @git+https://github.com/eclipse-uprotocol/up-transport-vsomeip-python.git@db101d4'
```
Comment (DivyaVuppala) [inline README.adoc]:
```suggestion
python3 -m pip install 'uprotocol-vsomeip-python @git+https://github.com/eclipse-uprotocol/up-transport-vsomeip-python.git@db101d4'
```

Comment (DivyaVuppala) [reply README.adoc]: Library name is "uprotocol-vsomeip-python" and not "up-transport-vsomeip-python"

Comment (neelam-kushwah) [reply README.adoc]: done

**Review by DivyaVuppala — APPROVED** [review]
SOMEIP Transport related changes looks good to me.

**Comment by neelam-kushwah** [comment]
Thanks @DivyaVuppala

**Pull request merged**